### PR TITLE
test: Add unit tests

### PR DESCRIPTION
**Pull request by nearora-msft**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/azurefile/nodeserver.go [hunk 1]
```diff
@@ -67,6 +67,7 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 			VolumeContext:     context,
 			VolumeCapability:  volCap,
 			VolumeId:          volumeID,
+			Secrets:           req.GetSecrets(),
```
Comment (andyzhangx) [inline pkg/azurefile/nodeserver.go:70]: what's this change for?

Comment (andyzhangx) [reply pkg/azurefile/nodeserver.go:70]: No, ephemeral volume does not need secret from node publish volume.

Comment (nearora-msft) [reply pkg/azurefile/nodeserver.go:70]: Ok I'll revert this

Comment (nearora-msft) [reply pkg/azurefile/nodeserver.go]: Done

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**